src/components/WwwFrame/TheHeader.vue

```diff
@@ -60,7 +60,7 @@
 							<img
 								:src="profilePic"
 								alt="My portfolio"
-								class="fs-mask"
+								class="fs-mask tw-inline-block"
```

Comment:
Do these classes actually work in our current main branch? The `kv-tailwind` class would have to be on a parent in the current state...they will work fine in the next version...

Reply:
This one was a autocomplete error